.github/workflows/verify.yml

```diff
@@ -2,13 +2,16 @@ on:
   pull_request_target:
     types: [opened, edited, reopened, synchronize]
 
+permissions:
+  checks: write # Allow access to checks to write check runs.
+
 jobs:
   verify:
     runs-on: ubuntu-latest
     name: verify PR contents
     steps:
     - name: Verifier action
       id: verifier
-      uses: kubernetes-sigs/kubebuilder-release-tools@v0.4.3
+      uses: kubernetes-sigs/kubebuilder-release-tools@012269a88fa4c034a0acf1ba84c26b195c0dbab4 # tag=v0.4.3
```

Comment:
Why not use the tags?

Reply:
These are pinned dependencies, to a hash. Versions might change the hash and that on its own has general security implications.

When updates are available, dependabot updates both the hash and the tag comment

Reply:
A while back dependabot didn't support updating tag comments, but since that's supported we've been using this in CAPI and it works perfectly